Author: Nathan Phillip Brink <ohnobinki@ohnopublishing.net>
Purpose: To fix compilation error when using LDFLAGS=-Wl,--as-needed
Uselessness: Not needed as of live-2009.09.28.

--- a/testProgs/Makefile.tail	2010-03-08 20:37:16.000000000 -0500
+++ b/testProgs/Makefile.tail	2010-03-08 20:37:27.000000000 -0500
@@ -59,7 +59,7 @@
 GROUPSOCK_DIR = ../groupsock
 GROUPSOCK_LIB = $(GROUPSOCK_DIR)/libgroupsock.$(LIB_SUFFIX)
 LOCAL_LIBS =	$(LIVEMEDIA_LIB) $(GROUPSOCK_LIB) \
-		$(USAGE_ENVIRONMENT_LIB) $(BASIC_USAGE_ENVIRONMENT_LIB)
+		$(BASIC_USAGE_ENVIRONMENT_LIB) $(USAGE_ENVIRONMENT_LIB)
 LIBS =			$(LOCAL_LIBS) $(LIBS_FOR_CONSOLE_APPLICATION)
 
 testMP3Streamer$(EXE):	$(MP3_STREAMER_OBJS) $(LOCAL_LIBS)

Diff finished.  Mon Mar  8 20:37:27 2010
--- a/mediaServer/Makefile.tail	2009-06-02 05:03:18.000000000 -0400
+++ b/mediaServer/Makefile.tail	2010-03-08 20:45:35.000000000 -0500
@@ -25,7 +25,7 @@
 GROUPSOCK_DIR = ../groupsock
 GROUPSOCK_LIB = $(GROUPSOCK_DIR)/libgroupsock.$(LIB_SUFFIX)
 LOCAL_LIBS =	$(LIVEMEDIA_LIB) $(GROUPSOCK_LIB) \
-		$(USAGE_ENVIRONMENT_LIB) $(BASIC_USAGE_ENVIRONMENT_LIB)
+		$(BASIC_USAGE_ENVIRONMENT_LIB) $(USAGE_ENVIRONMENT_LIB)
 LIBS =			$(LOCAL_LIBS) $(LIBS_FOR_CONSOLE_APPLICATION)
 
 live555MediaServer$(EXE):	$(MEDIA_SERVER_OBJS) $(LOCAL_LIBS)
--- a/Makefile.tail	2009-06-02 05:03:18.000000000 -0400
+++ b/Makefile.tail	2010-03-08 20:48:23.000000000 -0500
@@ -32,9 +32,9 @@
 	cd $(USAGE_ENVIRONMENT_DIR) ; $(MAKE)
 $(BASIC_USAGE_ENVIRONMENT_LIB):
 	cd $(BASIC_USAGE_ENVIRONMENT_DIR) ; $(MAKE)
-$(TESTPROGS_APP): $(LIVEMEDIA_LIB) $(GROUPSOCK_LIB) $(USAGE_ENVIRONMENT_LIB) $(BASIC_USAGE_ENVIRONMENT_LIB)
+$(TESTPROGS_APP): $(LIVEMEDIA_LIB) $(GROUPSOCK_LIB) $(BASIC_USAGE_ENVIRONMENT_LIB) $(USAGE_ENVIRONMENT_LIB)
 	cd $(TESTPROGS_DIR) ; $(MAKE)
-$(MEDIA_SERVER_APP): $(LIVEMEDIA_LIB) $(GROUPSOCK_LIB) $(USAGE_ENVIRONMENT_LIB) $(BASIC_USAGE_ENVIRONMENT_LIB)
+$(MEDIA_SERVER_APP): $(LIVEMEDIA_LIB) $(GROUPSOCK_LIB) $(BASIC_USAGE_ENVIRONMENT_LIB) $(USAGE_ENVIRONMENT_LIB)
 	cd $(MEDIA_SERVER_DIR) ; $(MAKE)
 
 clean:
